sql/2020/20_Caching/appcache_and_serviceworkers.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+# Use of AppCache and ServiceWorkers
+SELECT
+  SUBSTR(url, 0, 5) http_type,
+  JSON_EXTRACT_SCALAR(report, "$.audits.appcache-manifest.score") AS using_appcache,
+  JSON_EXTRACT_SCALAR(report, "$.audits.service-worker.score") AS using_serviceworkers,
+  count(*) AS occurences     
+FROM 
+  `httparchive.lighthouse.2020_08_01_mobile` 
+GROUP BY 
+  http_type, 
+  using_appcache, 
+  using_serviceworkers
+ORDER BY
+  http_type, 
+  using_appcache, 
+  using_serviceworkers

```

Comment:
Make the most prominent results easier to find at the top of the list.
```suggestion
  pct DESC
```